application/src/main/kotlin/net/corda/v5/application/marshalling/JsonSerializedBase64Config.kt

```diff
@@ -0,0 +1,12 @@
+package net.corda.v5.application.marshalling
```

Comment:
I wonder if this is the right place for this? I think at a minimum we should put it under its own package under `marshalling` to indicate that it's specific to JSON custom serializers. Possibly though it should be in its own module, as we might need to declare a dependency on this from the contract/data part of the cordapp, and that shouldn't really need the rest of the `application` API. Thoughts @mnesbit?

Reply:
Thinking about it a little more, I'd expect to only actually need the serializer when thinking about flows, so probably this is the right module for it at least.

Reply:
It's at the same level as JsonMarshallingService now. I can move it all of course. I wonder if we are going a bit package happy though, even Marshalling service mentions that its only subclass is JsonMarshallingService right now. I don't mind moving it all and removing that reference (this is a breaking change but that doesn't really matter) if that is what the consensus is?

Reply:
I've added @mnesbit to the review as well, probably should have done that anyway.